app/src/lib/git/commit.ts

```diff
@@ -4,6 +4,13 @@ import { Repository } from '../../models/repository'
 import { WorkingDirectoryFileChange } from '../../models/status'
 import { unstageAll } from './reset'
 
+/**
+ * Create a commit
+ *
+ * @param repository repository to execute merge in
+ * @param message commit message
+ * @param files files to commit
+ */
```

Comment:
Maybe this could return a string of the commit SHA instead of bool values.

Reply:
I really like this idea! I looked into doing this and it requires a bit of extra logic to parse the git output and get the SHA, so I'm going to make this into its own issue for the future. (https://github.com/desktop/desktop/issues/5923)